custom_components/versatile_thermostat/sensor.py

```diff
@@ -643,6 +643,14 @@ def __init__(self, hass: HomeAssistant, unique_id, name, entry_infos) -> None:
         self._attr_unique_id = "nb_device_active_boiler"
         self._attr_value = self._attr_native_value = None  # default value
         self._entities = []
+        self._attr_active_device_names = []  # Holds the names of active devices
+
+    @property
+    def extra_state_attributes(self) -> dict:
```

Comment:
Add the `active_device_names` into the list of `_entity_component_unrecorded_attributes` so that it will not be persisted and save place on the hard drive.

See base_thermostat.py at line 32 to have an exemple.

Reply:
Thank you for your feedback! Happy to adapt it :) 
But I am not sure if I can follow. [L32](https://github.com/jmcollin78/versatile_thermostat/blob/main/custom_components/versatile_thermostat/base_thermostat.py#L32) doesnt look right as an example.

Generally yes: we don't want to persist this on the hard drive.
My code extends the general configuration. 
So I was now looking to the list of `_entity_component_unrecorded_attributes` which is assigned to the central configuration. 
But all `_entity_component_unrecorded_attributes`  I could find are assigned to either the base thermostat (which hands down basic attributes and entities to all other climate entities) or the actual thermostats. 

(I am not a dev, so chances are pretty high, I don't understand the code ;) ) sorry :/